src/soter/openssl/soter_rsa_common.c

```diff
@@ -24,51 +24,66 @@
 #define SOTER_RSA_KEY_LENGTH 2048
 #endif
 
-soter_status_t soter_rsa_gen_key(EVP_PKEY_CTX* pkey_ctx)
+soter_status_t soter_rsa_gen_key(EVP_PKEY** ppkey)
 {
-    /* it is copy-paste from /src/soter/openssl/soter_asym_cipher.c */
```

Comment:
it's not anymore? :)

Reply:
> it's not anymore? :)

You know the story about the ship of Theseus?

Right now `soter_asym_cipher.c` does not even have generation in it. Whatever there was, this implementation does not really have much left in common with it, I think, so the comment only misleads the reader.